eng/pipelines/coreclr/templates/perf-job.yml

```diff
@@ -309,15 +309,15 @@ jobs:
             artifactFileName: 'iOSSampleAppSymbols.zip'
             artifactName: 'iOSSampleAppSymbols'
             displayName: 'iOS Sample App Symbols'
-      # same artifact as above but don't extract .zip
-      - task: DownloadBuildArtifacts@0
-        displayName: 'Download iOS Sample App Symbols'
-        inputs:
-          buildType: current
-          downloadType: single
-          downloadPath: '$(Build.SourcesDirectory)/iosHelloWorldZip/symbols'
-          artifactName: 'iOSSampleAppSymbols'
-          checkDownloadedFiles: true
+        # same artifact as above but don't extract .zip
+        - task: DownloadBuildArtifacts@0
+          displayName: 'Download iOS Sample App Symbols'
+          inputs:
+            buildType: current
+            downloadType: single
+            downloadPath: '$(Build.SourcesDirectory)/iosHelloWorldZip/symbolszip'
+            artifactName: 'iOSSampleAppSymbols'
```

Comment:
Might not be that relevant and I am not sure at which point these names are visible, but just wondering whether there should be a distinction between MonoAOT and NativeAOT in these names and paths, to avoid confusion.

Reply:
Thanks for pointing out, the code has been simplified 🚀